### PR TITLE
gk-deploy: Error and Exit when heketi-cli commands failed

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Closes: #104 
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>
Pair Programmed with: Ramakrishna Reddy Yekulla <rreddy@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/121)

### CODE REVIEW

File: deploy/gk-deploy [hunk 1]
```diff
@@ -482,8 +482,30 @@ if [[ "${hello}" != "Hello from Heketi" ]]; then
   abort
 fi
 
-heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY}
+tload=$(heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY} 2>&1)
+if [[ ${?} != 0 ]]; then
+	echo $tload
```
Comment (raghavendra-talur) [inline deploy/gk-deploy]: Use output instead of echo

Comment (raghavendra-talur) [inline deploy/gk-deploy]: Use standard indentation of 2 spaces

File: deploy/gk-deploy [hunk 2]
```diff
@@ -482,8 +482,30 @@ if [[ "${hello}" != "Hello from Heketi" ]]; then
...
+	exit 1
+else
+        check=""
+	echo $tload
+        check=$(echo $tload | grep Unable)
```
Comment (raghavendra-talur) [inline deploy/gk-deploy]: You could use quite option of grep. Also consider -i of grep.

File: deploy/gk-deploy [hunk 3]
```diff
@@ -482,8 +482,27 @@ if [[ "${hello}" != "Hello from Heketi" ]]; then
   abort
 fi
 
-heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY}
+tload=$(heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY} 2>&1)
+echo "${tload}"
+if [[ ${?} != 0 ]]; then
```
Comment (raghavendra-talur) [inline deploy/gk-deploy]: you would be ret value of echo here, not heketi-cli.

**Comment by humblec** [comment]
LGTM.

**Pull request merged**